### PR TITLE
Patricia Manalang - completed css-four-column-layout exercise.

### PR DESCRIPTION
**Pull request by patriciamanalang**
Screen Shot 2022-10-17 at 9 34 45 PM

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
+.container {
+  margin: auto;
+  max-width: 80%;
+  background-color: rgb(247 248 250);
+  padding-left: 50px;
+  padding-right: 50px;
+  padding-bottom: 13px;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css]: Only `margin` and `max-width` on the container, class, please shift these declarations to another class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by patriciamanalang** [comment]
Screen Shot 2022-10-18 at 8 59 08 PM

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -13,12 +9,17 @@
 
 .column-one-fourth {
   flex-basis: 25%;
+  background-color: rgb(247 248 250);
+  padding-left: 50px;
+  padding-right: 50px;
+  padding-bottom: 13px;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css]: Only `flex-basis` should be applied to `column-` classes. Make another class that you can add to the element in addition to the `column-one-fourth` class.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by patriciamanalang** [comment]
Screen Shot 2022-10-20 at 12 27 44 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**